pipestat/pipestat.py

```diff
@@ -118,56 +118,70 @@ def __init__(
         """
 
         super(PipestatManager, self).__init__()
-        self.store = dict()
+
+        # Initialize the initial dict
+        self.cfg = {}
 
         # Load and validate database configuration
         # If results_file_path is truthy, backend is a file
         # Otherwise, backend is a database.
-        self._config_path = select_config(config_file, ENV_VARS["config"])
-        _LOGGER.info(f"Config: {self._config_path}.")
-        self[CONFIG_KEY] = YAMLConfigManager(entries=config_dict, filepath=self._config_path)
+        # self._config_path = select_config(config_file, ENV_VARS["config"])
+
+        self.cfg["config_path"] = select_config(config_file, ENV_VARS["config"])
+
+        # _LOGGER.info(f"Config: {self.cfg["config_path"]}.")
+
+        self.cfg[CONFIG_KEY] = YAMLConfigManager(
+            entries=config_dict, filepath=self.cfg["config_path"]
+        )
+
         _, cfg_schema = read_yaml_data(CFG_SCHEMA, "config schema")
-        validate(self[CONFIG_KEY].exp, cfg_schema)
+        validate(self.cfg[CONFIG_KEY].exp, cfg_schema)
 
-        self[SCHEMA_PATH] = self[CONFIG_KEY].priority_get(
+        self.cfg[SCHEMA_PATH] = self.cfg[CONFIG_KEY].priority_get(
             "schema_path", env_var=ENV_VARS["schema"], override=schema_path
         )  # schema_path if schema_path is not None else None
         self.process_schema(schema_path)
 
-        self[RECORD_IDENTIFIER] = record_identifier
+        # self[RECORD_IDENTIFIER] = record_identifier
```

Comment:
Can you please clean this. to many comments are. And probably noone will use them